src/App.tsx

```diff
@@ -56,8 +68,18 @@ export function App() {
     }
   }, [darkMode, resolvedLanguage]);
 
+  useEffect(() => {
+    if (user) {
+      setIsEmailVerified(!!user.email_verified_at);
```

Comment:
Please don't use double !!, kittens will die. 

Stick to Boolean as it's more readable.

Reply:
changed